app/konnect/configure/runtime-manager/runtime-groups/manage.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Manage Runtime Groups
+no_version: true
+---
+
+Create, update, and delete runtime groups through the
+[Runtime Manager](https://konnect.konghq.com/runtime-manager) in Konnect.
+
+You can find the list of all runtime groups in the Runtime Manager. If you don't
+have an Enterprise account, the Runtime Manager will only ever have one
+runtime group entry.
+
+From a runtime group details page, you can
+[create and manage runtime instances](/konnect/configure/runtime-manager/#types-of-runtimes)
+and configure any [global entities](/konnect/configure/runtime-manager/manage-entities)
+in the group.
+
+## Prerequisites
+* TBA: Required permissions
+* TBA: Any prereqs for deleting the group
+
+## Create a runtime group
+{:.badge .enterprise}
+
+1. In Konnect, open the ![runtimes icon](/assets/images/icons/konnect/icn-runtimes.svg){:.inline .konnect-icn .no-image-expand}
+**Runtime Manager** from the left side menu.
+
+1. Click **Create Runtime Group**.
+
+1. Name the runtime group and enter an optional description.
+
+    Each runtime group in the organization must have a unique name.
+
+1. Add any labels in key-value pair format.
```

Comment:
@haydenlam I notice that the description and label fields have disappeared from cloud-02. Are they descoped from helium?

Reply:
The work is still being done, just on a lower priority than the others.
I'm expecting it to show up in the next 1-2 weeks, so I'm not opposed to leaving it in the docs. The overview panel currently shows mock data in the labels attribute. (But edit UI is not ready)

Reply:
Since we don't have a concrete timeline for this, and it won't be in before I merge this PR, I'm removing it for now. We can add it back in when the UI matches expectations.